src/backend/marsha/core/templates/core/lti_development.html

```diff
@@ -241,7 +241,7 @@ <h2>Iframe LTI Resource setup</h2>
             <option value="documents">document</option>
             <option value="classrooms">classroom</option>
             <option value="markdown-documents">Markdown document</option>
-            <option value="filedepositories">File depository</option>
+            <option value="deposits">File depository</option>
```

Comment:
You also need to update the `marsha/src/backend/marsha/development/views.py` file to return the last objects dict with proper keys:
(note: the markdown document key must also be fixed)
```
            "last_objects": {
                "videos": Video.objects.order_by("-updated_on")[:5],
                "documents": Document.objects.order_by("-updated_on")[:5],
                "classrooms": Classroom.objects.order_by("-updated_on")[:5],
                "markdown-documents": MarkdownDocument.objects.order_by("-updated_on")[
                    :5
                ],
                "deposits": FileDepository.objects.order_by("-updated_on")[:5],
            },
```

Reply:
Good catch :+1: